src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -29,13 +33,16 @@ export default function * fetchResourceCache (city: string, language: string, fe
     const targetUrls = mapValues(fetchMap, ([url]) => url)
 
     if (Platform.OS !== 'android') {
-      return {failureMessages: {}, fetchedUrls: {}}
+      return {}
     }
 
-    const result: FetchResultType = yield call(new FetcherModule().fetchAsync, targetUrls, progress => {})
+    const results: FetchResultType = yield call(new FetcherModule().fetchAsync, targetUrls, progress => {})
+
+    const successResult = pickBy(results, result => !result.errorMessage)
```

Comment:
successResult*s*

Reply:
The type has changed to a single map. Error results and Sucess results are now in one map.